Assets/Scripts/Bahavior/Dash.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Dash : MovementBehavior
+{
+    [Header("Animations")]
+    [SerializeField] GameObject spriteObject;
+    [SerializeField] private Animator anim;
+    [Header("Stats")]
+    [SerializeField] float dashPower;
+    [SerializeField] float dashTime;
+    [SerializeField] float dashCoolDown;
+
+    //Must-have variables for movements
+    [SerializeField] IDashInput dashInput;
+    [SerializeField] TrailRenderer tr;
+
+    [SerializeField] Transform groundCheck;
+    [SerializeField] float checkRadius;
+    [SerializeField] LayerMask whatIsGround;
+
+    private bool isGrounded;
+
+    // Start is called before the first frame update
+    void Start()
```

Comment:
Override Start() function from MovementBehavior class and then call the base.Start() function.

Reply:
Oge